README.md

```diff
@@ -108,6 +113,7 @@ npm install --save-dev babel-plugin-inline-svg
 - _`disableSVGO`_ - set to `false` to disable running the svg through SVGO
 - _`disableNamespaceIds`_ - set to `false` to leave all id's as they are
 - _`svgo`_ - an object of SVGO options
+- _`exportDataURI`_ - set to `true` to export encoded as base64 and prefixed with `data:image/svg+xml;base64,`
```

Comment:
Yeah I think you're right, this name works better. I would suggest some slight wording tweaks:

```suggestion
- _`exportDataURI`_ - set to `true` to export a base64-encoded SVG, prefixed with `data:image/svg+xml;base64,`
```